package.json

```diff
@@ -91,5 +95,6 @@
     "realtime",
     "content",
     "image-url"
-  ]
+  ],
+  "packageManager": "pnpm@10.15.0"
```

Comment:
The `release.yml` reusable action requires `pnpm` to be used, and `scripts.release` to have `changeset publish`.